### PR TITLE
Git issue template for new assets addition

### PR DESCRIPTION
**Pull request by LefterisJP**

### CODE REVIEW

**Comment by yabirgb** [comment]
Aren't we go to stick with an issue per release?

File: .github/ISSUE_TEMPLATE/support_new_asset.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+name: Support new asset
+about: Request support for a new asset
+---
+
+<!-- You can request for official support of an asset by filling in this template. But you can also easily add support for it on your local copy by following this guide: https://rotki.readthedocs.io/en/latest/usage_guide.html#adding-editing-an-asset -->
+
+
+**Asset name:** Please fill in
+**Asset symbol:** Please fill in
+**Coingecko link:** Please fill in
```
Comment (yabirgb) [inline .github/ISSUE_TEMPLATE/support_new_asset.md:11]:
```suggestion
**Coingecko link:** Please fill in - if available
```

**Comment by LefterisJP** [comment]
> Aren't we go to stick with an issue per release?

This is to help people who open all these issues have a standardized way to report to us, or even avoid reporting all together, by learning they can add it locally.

File: .github/ISSUE_TEMPLATE/support_new_asset.md (hunk 1 above)
Comment (LefterisJP) [reply .github/ISSUE_TEMPLATE/support_new_asset.md:11]: If that's not available then the asset will be rejected by us.

Comment (yabirgb) [reply .github/ISSUE_TEMPLATE/support_new_asset.md:11]: But it's enough if it has one of both (coingecko/cc), doesn't have to be coingecko

Comment (LefterisJP) [reply .github/ISSUE_TEMPLATE/support_new_asset.md:11]: fair enough. True.

**Review by yabirgb — APPROVED** [review]

**Pull request merged**